fe/example/src/App/App.treat.ts

```diff
@@ -0,0 +1,9 @@
+import { globalStyle } from 'sku/treat';
+
+globalStyle('body', {
+  height: '100vh',
+});
+
+globalStyle('#app, #app > div, #app > div > div', {
```

Comment:
This structure should not change, so I think it's fine. However, these nested styles could hurt us down the track. There should be ways to isolate/ attach the styles to the correct divs to push the height to 100vh.

Reply:
Yeah this is gross, I'll have a look

Reply:
That works a treat! Ryan always gets salty about 100vh breaking his mobile Safari though

Reply:
Mobile Safari is living in a hell it created for itself. Let it burn.